### DISCUSSION TITLE
save UTF-8 characters

### DISCUSSION THREAD

**Discussion by andretk99** [spatie/laravel-activitylog#911] [General] [answered]
I can't get this function to save UTF-8 characters as expected.

Dato nuevo:{"nombre":"a\u00f1o"},Dato antiguo:{"nombre":"ani\u00f3"}}
someone who has had this same problem, could help me

**Comment by Gummibeer** [comment, marked as answer]
That's correctly saved as the character is restored while retrieving the record.

**Reply by Gummibeer** [reply]
#865 #522 #351
